### ISSUE TITLE
sqlTextPreviewMaxLength parameter doesn't change the actual limit 

### ISSUE THREAD

**Issue by AviSarmani** [dbeaver/cloudbeaver#1562]
**Describe the bug**
sqlTextPreviewMaxLength parameter doesn't allow to show longer texts even when changed in server conf file

**To Reproduce**
1. Change the parameter sqlTextPreviewMaxLength to a bigger one that the default(8192)
2. Go to a table that has a large text field
3. Click on the row and open the value panel
4. See that the value is still limited to 4096 characters

**Screenshots**
The issue:
![image](https://user-images.githubusercontent.com/49937628/227176358-822adcf5-6c10-4223-bd4c-b3763ea9f65d.png)

Server conf file:
![image](https://user-images.githubusercontent.com/49937628/227175429-2f7ea3d4-ba6a-423c-985b-436315a4c64d.png)

Version:
![image](https://user-images.githubusercontent.com/49937628/227175625-4ae3f3cc-7dad-4829-8ef9-5b8eaa14e22e.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome
 - Version 23.0.0.

**Comment by dariamarutkina** [comment]
Hello @AviSarmani ,
please try to change this setting (sqlTextPreviewMaxLength) in the file [`.cloudbeaver.runtime.conf` ](https://github.com/dbeaver/cloudbeaver/wiki/Server-configuration) 
Thank you for your interest in CloudBeaver.

**Comment by AviSarmani** [comment]
Hi @dariamarutkina ,
Thank you for the reply.
The .cloudbeaver.runtime.conf file gets updated together with conf/cloudbeaver.conf
Unfortunately the issue still persists.
I tried adding the "dataExportFileSizeLimit" to see if it's being used and looks like it works as expected so it seems to be a specific issue with "sqlTextPreviewMaxLength" parameter
Upgraded to 23.0.1 and didn't help as well :(
![image](https://user-images.githubusercontent.com/49937628/228175017-8c07a6bd-61f0-4bfe-87e5-0547ef3484ad.png)
conf/cloudbeaver.conf:
```
{
    server: {
        serverPort: 8978,
        workspaceLocation: "workspace",
        contentRoot: "web",
        driversLocation: "drivers",
        rootURI: "/cloudbeaver/",

        serviceURI: "/api/",

        productConfiguration: "conf/product.conf",

        expireSessionAfterPeriod: 3600000,

        develMode: false,

        database: {
            driver="h2_embedded",
            url: "jdbc:h2:${workspace}/.data/cb.h2.dat",

            createDatabase: true,

            initialDataConfiguration: "conf/initial-data.conf",

            pool: {
                minIdleConnections: 4,
                maxIdleConnections: 10,
                maxConnections: 100,
                validationQuery: "SELECT 1"
            }
        }

    },
    app: {
        anonymousAccessEnabled: false,
        anonymousUserRole: "user",
        supportsCustomConnections: false,

        publicCredentialsSaveEnabled: true,
        adminCredentialsSaveEnabled: true,
        resourceQuotas: {
        sqlTextPreviewMaxLength: 8192,
        dataExportFileSizeLimit: 100
            }
    }
}
```
workspace/.data/.cloudbeaver.runtime.conf:
```
{
  "server": {
    "serverName": "my-cloudbeaver",
    "expireSessionAfterPeriod": 3600000,
    "database": {
      "pool": {
        "minIdleConnections": 4.0,
        "maxIdleConnections": 10.0,
        "maxConnections": 100.0,
        "validationQuery": "SELECT 1"
      },
      "createDatabase": true,
      "initialDataConfiguration": "conf/initial-data.conf",
      "driver": "h2_embedded",
      "url": "jdbc:h2:/opt/cloudbeaver/workspace/.data/cb.h2.dat"
    }
  },
  "app": {
    "anonymousAccessEnabled": false,
    "supportsCustomConnections": false,
    "publicCredentialsSaveEnabled": true,
    "adminCredentialsSaveEnabled": true,
    "enableReverseProxyAuth": false,
    "forwardProxy": false,
    "linkExternalCredentialsWithUser": true,
    "redirectOnFederatedAuth": false,
    "resourceManagerEnabled": true,
    "showReadOnlyConnectionInfo": false,
    "grantConnectionsAccessToAnonymousTeam": false,
    "resourceQuotas": {
      "sqlTextPreviewMaxLength": 8192.0,
      "dataExportFileSizeLimit": 100.0
    },
    "defaultNavigatorSettings": {},
    "enabledFeatures": [],
    "enabledAuthProviders": [
      "local",
      "reverseProxy"
    ],
    "enabledDrivers": [],
    "disabledDrivers": []
  }
}
```

**Comment by dariamarutkina** [comment]
Hello @AviSarmani ,
Thank you for the report. I've reproduced the bug. We will fix this in the future.
Thank you for your interest in Cloud Beaver.

**Comment by TatyanaSsau** [comment]
Fixed in 23.0.4

@AviSarmani, thank you for interesting in CloudBeaver!
